common/web/lm-worker/src/main/correction/context-tracker.ts

```diff
@@ -407,41 +373,81 @@ export class ContextTracker extends CircularArray<TrackedContextState> {
       }
     }
 
-    const hasDistribution = transformDistribution && Array.isArray(transformDistribution);
-    let primaryInput = hasDistribution ? transformDistribution[0].sample : null;
-    if(primaryInput && primaryInput.insert == "" && primaryInput.deleteLeft == 0 && !primaryInput.deleteRight) {
-      primaryInput = null;
-    }
-
-    // TODO:  "wordbreak" the `insert` section of the transform (if it exists).
-    // ... wait, might have to be done at a higher level...
-    // ... and will probably want its own unit test ...
-
-    const isBackspace = primaryInput && TransformUtils.isBackspace(primaryInput);
+    const hasDistribution = transformSequenceDistribution && Array.isArray(transformSequenceDistribution);
 
     // Reset priorEdit for the end-of-context updating loop.
     priorEdit = undefined;
 
+    // TODO:  add flag:  rebuiltSearchSpace (?)
+    //
+    // Or should searchSpace be tracked (eventually) on the tokens, rather than on the 'state'?
+    // - I'm leaning toward the latter here.
+    // - Shifting the paradigm should be a separate work unit than the context-tracker rework
+    //   currently being done, though.
```

Comment:
These are notes toward potential future work - things that would need to be addressed to support phrase-level corrections.